content/technotes/2019-01-10-hugo.md

```diff
@@ -0,0 +1,101 @@
+---
+slug: hugo
+title: An even better rOpenSci website? Off Hugo!
+date: '2019-01-10'
+topicid:
+authors:
+  - Maëlle Salmon
+categories: technotes
+tags:
+  - hugo
+---
+
+A bit more than one year ago, rOpenSci [launched its new website design](https://twitter.com/rOpenSci/status/925036483383148545), by the designer [Maru Lango](https://www.marulango.com/). Not only did the website appearance change (for the best!), but the underlying framework too. ropensci.org is powered by [Hugo](https://gohugo.io/), like `blogdown`! Over the last few months, we've made the best of this framework, hopefully improving your browsing experience (and trapping you into binge reading). In this note, we'll go over the main developments, as well as give some Hugo tips. 
+
+# Our website and its source
+
+Skip this section if you just want to know what's become better on our website!
+
+Our website is located at [ropensci.org](https://ropensci.org). Its source lives on GitHub in [the ropensci/roweb2 repo](https://github.com/ropensci/roweb2/). The website is deployed on Netlify from the master branch, and each pull request gets a Netlify preview link too, which is quite handy when reviewing new blog posts.
```

Comment:
"deployed on Netlify" is not clear - does that mean they're the people who host it (rather the gh pages), the place it's built, or somrthing else?